cmake/chai-config.cmake.in

```diff
@@ -0,0 +1,31 @@
+##############################################################################
+# Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and CHAI
```

Comment:
Do you folks update the copyright dates at the beginning of each new year?

Reply:
I can take a look at the script in RAJA.

Reply:
It's pretty manual, but straightforward. If you have any questions please let me know. Depending on how inconsistent the dates are in the files it may require running it over the code multiple times with different sed options.

Reply:
https://github.com/LLNL/CHAI/pull/248